src/components/node-list/node-list-row.test.js

```diff
@@ -85,6 +84,37 @@ describe('NodeListRow', () => {
         false
       );
     });
+
+    describe('focus mode', () => {
+      it('sets the visibility toggle to the checked mode when the row is selected for focusmode', () => {
+        const { props } = setupProps();
+        const wrapper = setup.mount(
+          <NodeListRow {...props} focusMode={node} type="modularPipeline" />
```

Comment:
Nit: As a public props API, I do find `focusMode={node}` a bit hard to parse. The prior art of `nodeSelected={node}` makes more sense to me. A suggestion is `nodeFocused={node}`.